p11mod/p11mod.go

```diff
@@ -222,9 +222,20 @@ func (ll *llBackend) GetMechanismInfo(slotID uint, m []*pkcs11.Mechanism) (pkcs1
 }
 
 func (ll *llBackend) InitPIN(sh pkcs11.SessionHandle, pin string) error {
-	// TODO
-	log.Println("p11mod InitPIN: not implemented")
-	return pkcs11.Error(pkcs11.CKR_FUNCTION_NOT_SUPPORTED)
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return err
+	}
+
+	if trace {
+		log.Printf("p11mod initpin")
```

Comment:
Please capitalize "InitPIN" to match the function name.

Reply:
Done ✅